static/scss/pages/home.scss

```diff
@@ -0,0 +1,60 @@
+.mzp-t-relay-home {
+    background-color: transparent;
+    text-align: center;
+    
+    .mzp-l-content {
+        max-width: $content-lg;
+        margin-left: auto;
+        margin-right: auto;
+        padding-top: $layout-xl;
+        padding-bottom: $layout-xl;
+    }
+
+    .mzp-c-hero-desc {
+        max-width: $content-md;
+        margin-left: auto;
+        margin-right: auto;
+    }
+
+    .mzp-c-hero-title {
+        @include text-title-md;
+        background: var(--h1Gradient);
+        background-clip: text;
+        -webkit-background-clip: text;
+        -webkit-text-fill-color: transparent;
+    }
+
+    .mzp-c-button {
+        min-width: 100%;
+
+        @media #{$mq-sm} {
+            min-width: $layout-2xl;
+        }
+    }
+}
+
+.c-hero-slides {
+    background-color: $color-white;
+    
+    div {
+        margin-bottom: $spacing-2xl;
+
+        @media #{$mq-md} {
+            margin-bottom: 0;
+        }
+    }
+
+    h2 {
+        @include text-title-3xs;
+        color: $color-purple-90;
+        text-align: center;
+        margin-bottom: $spacing-md;
+    }
+
+    img {
+        display: block;
+        width: $content-xs;
+        margin-left: auto;
```

Comment:
nit: `margin: 0 auto;`?

Reply:
Revised. I had an additional commit that retooled a ton of the "lines" of the site with the `$content-max` width set everything. I ended up bailing on it, but I was constantly only using `margin-left/right` to preserve the Y axis margins from a parent style. 

TLDR; Great catch!